### DISCUSSION TITLE
[CollaborationInfo/AgreementRef/@type must not be set]

### DISCUSSION THREAD

**Discussion by yvansteene** [phax/phase4#253] [General]
Hi Philip,

Since version 5.1.5, the Domibus platform enforces the type attribute on the pmode configuration. 
Someone tried to send us some documents, but this verification on our phase4 AP blocks the reception. 
I don't know what to do. The Peppol AS4 profile documentation confirms that it must be without the type attribute.
Can I accept the document if this attribute is set?

I'm mentioning it here because it could affect other people using phase4.

Thank you for your help and all your work.
Vansteene Yoan

**Comment by phax** [comment]
Dear Yoan,
Thanks for the reaching out to me and bringing that to my attention.
The Peppol AS4 profile 2.0.3 chapter 4.3. is what you are referring to: https://docs.peppol.eu/edelivery/as4/specification/#_agreement
> ... MUST have value `urn:fdc:peppol.eu:2017:agreements:tia:ap_provider` without `type` attribute.

I assume we should file a bug report in Domibus....

Edit: I was looking through the changes from 5.1.4 to 5.1.5 (https://ec.europa.eu/digital-building-blocks/tracker/issues/?jql=project+%3D+EDELIVERY+AND+fixVersion+%3D+%22Domibus+5.1.5%22 - you need an EC user to access it) and couldn't find the change you mentioned. So eventually it was enforced even in an earlier release???

Thanks, Philip

**Reply by yvansteene** [reply]
Dear Philip ,

sorry, i don't have access to Domibus. The only information I have is that someone who used to send us documents can no longer contact us since they upgraded to version 5.1.5. 
They told me that the type attribute is now mandatory in Domibus.
I can't test by myself this information either

I can contact him to ask for information about which version he used before.

Thanks, Yoan

**Reply by yvansteene** [reply]
We found a way to avoid filling the "type" attribute in Domibus by putting a " " as its value. 
For your information, they upgraded from 5.1.0 to 5.1.5.

Thanks, Yoan

**Reply by phax** [reply]
Thanks for confirming that and thanks for providing a workaround here. Highly appreciated.

**Reply by phax** [reply]
Still waiting for feedback from the EC tec team on the specific issue. Lets wait and see

**Reply by yvansteene** [reply]
Thanks for the update . We haven't received any further feedback from our client, so I assume the workaround is working.

**Reply by phax** [reply]
Here's a statement from EC support:
> The workaround is to set the agreement type to an empty value in the Pmode eg type="". The type attribute will be set optional one of the future bug fixes.

That's all I know.

**Reply by yvansteene** [reply]
Thanks
